deploy/kustomize/manifests/image_registry/catalog_source.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-catalogsource
+  namespace: openshift-marketplace
+  imagePullPolicy: Always
+spec:
+  sourceType: grpc
+  image: quay.io/kubevirt/hco-container-registry:latest
+  displayName: KubeVirt HyperConverged
+  publisher: Red Hat
```

Comment:
Kubevirt ?

Reply:
In current script the publisher is Red Hat:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/deploy_imageregistry.sh#L59
You suggest to change the publisher to `kubevirt` ?